java/change-notes/2020-10-03-android-intent-taintsource.md

```diff
@@ -0,0 +1,3 @@
+lgtm,codescanning
+* Exported Android `Intent`s have been added as sources of tainted data for all
```

Comment:
Maybe be more specific?
```suggestion
* Exported Android `Intent`s have been added as remote sources of tainted data for all
```

Reply:
I know that we have some modelling of local taint sources, but these are not really used in practice - they merely exist to support certain benchmark tests. In practice all sources of tainted data are remote sources, so I'm not sure the distinction is worth pointing out here - I think we'd rather be succinct, but in the end I'll leave that to our documentation team.